assemblies/dist-allinone/src/main/resources/bin/setenv

```diff
@@ -38,3 +38,33 @@ export EXTRA_JAVA_OPTS="-Dorg.eclipse.jetty.server.Request.maxFormContentSize=15
 export JAVA_MAX_MEM=1G
 export JAVA_PERM_MEM=256M
 export KARAF_NOROOT=true
+
+
+# -----------------------------------------------------------------------
+# check if http or https port is available to run opencast otherwise exit
+
+usehttp=`grep "org.osgi.service.http.enabled[ =]" etc/org.ops4j.pax.web.cfg`
+if [ ${usehttp##*=} = "true" ]; then 
+  httpport=`grep "org.osgi.service.http.port[ =]" etc/org.ops4j.pax.web.cfg`
+  if [ -x /bin/netstat ]; then
```

Comment:
Two remarks:

* Could this be located in another location (e.g. `/usr/bin`)? Probably better to check with `[ -x "$(command -v netstat)" ]` to use the `$PATH`.
* The Docker images currently do not include `netstat`. Not a problem there as the container interface should never have port conflicts, but can we assume that `netstat` is usually installed?

Reply:
If netstat is not installed we fall trough to the current behaviour, so we try to start opencast anyway